tensorflow/python/eager/benchmarks_test.py

```diff
@@ -1620,5 +1627,42 @@ def benchmark_tf_range_return_int32_GPU(self):
   def benchmark_tf_range_return_int64_GPU(self):
     self._benchmark_tf_range_return(dtype=dtypes.int64, device=GPU)
 
+  def _benchmark_embedding_lookup_sparse_sparse_input(self, batch_size=32000, device=GPU):
+    def func(sp_ids):
+      return embedding_ops.embedding_lookup_sparse(self._m_10000_by_16, sp_ids, None)
+
+    with context.device(device):
+      values = random_ops.random_uniform(shape=(batch_size,),
+                        minval=1,
+                        maxval=10000,
+                        dtype=dtypes.int64)
+      value_rowids = ops.EagerTensor(np.arange(batch_size), device=device)
+
+      ragged_input = ragged_tensor.RaggedTensor.from_value_rowids(values, value_rowids)
+      sparse_input = sparse_ops.from_dense(ragged_input.to_tensor())
```

Comment:
RaggedTensor has a method to convert itself to a SparseTensor: `spase_input = ragged_input.to_sparse()`